packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -749,6 +749,12 @@ export async function getOrganization(req: AuthRequest, res: Response) {
     effectiveAccountPlan: getEffectiveAccountPlan(org),
     licenseError: getLicenseError(org),
     commercialFeatures: [...accountFeatures[getEffectiveAccountPlan(org)]],
+    fullCommercialFeaturesMap: Object.fromEntries(
+      Object.entries(accountFeatures).map(([plan, features]) => [
+        plan,
+        Array.from(features),
+      ])
+    ),
```

Comment:
I might recommend doing the pre-processing on the back-end so we just send down a map from feature to the minimum plan required.  Will make it easier to manage if that logic is in one place.

Reply:
Yeah I thought about doing it but wasn't sure I wanted to make "lowest plan per feature" a canonical mapping outside of a component since "lowest plan" is debatable.